files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -177,16 +177,10 @@
   [[ else ]]
   - role: node
   [[ end ]]
-[[ if ne .ClusterType "management_cluster" ]]
-    api_server: https://[[ .APIServerURL ]]
-    tls_config:
-      ca_file: /etc/prometheus/secrets/[[ .SecretName ]]/ca
-      cert_file: /etc/prometheus/secrets/[[ .SecretName ]]/crt
-      key_file: /etc/prometheus/secrets/[[ .SecretName ]]/key
-      insecure_skip_verify: false
-[[ else ]]
+[[ include "_apiserver" . ]]
+[[- if eq .ClusterType "management_cluster" ]]
   bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
-[[ end ]]
```

Comment:
why these 3 above lines are not embedded in the _apiserver section ?

Reply:
That's because etcd is a bit special, especially on vintage

Reply:
Also the apiserver secion is only for workload clusters because prometheus in the MC have access to the api server "locally"

Reply:
On MC there is always a file with the token ?